streams/src/test/java/org/apache/kafka/streams/processor/internals/StoreChangelogReaderTest.java

```diff
@@ -307,15 +284,16 @@ public void shouldSupportUnregisterChangelogBeforeCompletion() {
     public void shouldSupportUnregisterChangelogAfterCompletion() {
         setupStateManagerMock();
         setupStoreMetadata();
+        setupStore();
+        @SuppressWarnings("unchecked")
```

Comment:
This is required because otherwise mocking a Map throws a tantrum. However, if there are nicer ways to achieve the same I am more than happy to change it